src/abstractinterpret/inferenceerrorreport.jl

```diff
@@ -258,25 +274,30 @@ function handle_sig_call!(sig::Vector{Any}, s::StateAtPC, @nospecialize(f), args
     splat::Bool = false)
     handle_sig!(sig, s, f)
     push!(sig, '(')
+    @show f typeof(f)
+    typs = Any[typeof_sig_f(s, f)]
+    @show typs
     nargs = length(args)
     for (i, arg) in enumerate(args)
         handle_sig!(sig, s, arg)
+        push!(typs, sig[end])
```

Comment:
`sig[end]` can be arbitrary IR element, so it would be better to use `typeof_sig_f(s, arg)` here?